### ISSUE TITLE
Project health clarification

### ISSUE THREAD

**Issue by stevespringett** [microsoft/OSSGadget#27]
I've noticed that some projects have 0.0% for issues or security.

Is this due to a lack of historical data in which to determine health? If so, then what would be an alternative way to display this info as the lack of evidence seems to imply a project is doing poorly which is not the case. And likewise, a lack of evidence should not necessarily be justification to make the project look good either.

Thoughts?

**Comment by scovetta** [comment]
Yes, thank you @stevespringett - I agree, neither 0% nor 100% convey what we want to say. I suppose one could ask, which project would you prefer, one that had three security issues, all promptly fixed, or one that had no security issues?
